src/Dialogs/Message.cpp

```diff
@@ -153,3 +153,77 @@ ShowMessageBox(const TCHAR *text, const TCHAR *caption, unsigned flags)
 
   return res;
 }
+
+
+
+int    //////// LOCKSCREEN
+ShowLockBox()
+{
+
+  SingleWindow &main_window = UIGlobals::GetMainWindow();
+
+  const unsigned button_width = Layout::Scale(15u);
+  const unsigned button_height = Layout::Scale(15u);
+
+  // Create dialog
+  WindowStyle style;
+  style.Hide();
+  style.ControlParent();
+
+  const DialogLook &dialog_look = UIGlobals::GetDialogLook();
+
+  PixelRect form_rc;
+  form_rc.left = 0;
+  form_rc.top = 0;
+  form_rc.right = button_width;
+  form_rc.bottom = button_height;
+
+  WndForm wf(main_window, dialog_look, form_rc, "", style);
```

Comment:
This doesn't compile because the `WndForm` constructor expects a `const TCHAR*` but you pass a `const char *`.